include/cpuinfo_x86.h

```diff
@@ -140,6 +144,10 @@ typedef enum {
   X86_SMX,
   X86_SGX,
   X86_CX16,
+  X86_SHA,
+  X86_POPCNT,
+  X86_MOVBE,
+  X86_RDRND,
```

Comment:
prefer to use `RDRAND` like the ANDROIR macro.
note: wikipedia seems to use `RdRand` https://fr.wikipedia.org/wiki/RdRand

Reply:
I've used the naming from https://en.wikipedia.org/wiki/CPUID for the field in the `X86Features` struct and I wanted to make sure the enums and the field names are spelled the same way - since they need to stay in sync. What do you think?

Reply:
So if you don't mind I'll proceed and submit this patch as-is.
Let me know what you think by commenting here and I'll fix it in a subsequent PR.